### ISSUE TITLE
Disable `Connect Wallet` until needed connection is established

### ISSUE THREAD

**Issue by traumschule** [Joystream/pioneer#3804]
## Scope
- [ ] Allow user to start the onboarding process even if the chain is not connected. 
- [ ] Disable Join Now if the chain is not connected

## Context

if user is not connected with the chain they cannot see the onboarding modal. 

=> sidebar

<img width="444" alt="Screenshot 2022-11-08 at 14 58 16" src="https://user-images.githubusercontent.com/93924525/200547313-1313c7f4-bb86-40a2-b5ec-d43f9dd872cc.png">

**Comment by dmtrjsg** [comment]
@traumschule why would you want this? we think it may make sense to disable "join now" button before the connection is established to show memberships but no such problem with the wallet.. wdyt?

**Comment by traumschule** [comment]
Right, lacked some detail here. The problem is the button does nothing until endpoint connection is established. So 
1. Add/Verify logic to indicate if buttons are already usable or not. Mostly already in place.
2. On top of that `Connect wallet` would have to be disabled until api connection and it may not be completely clear to new users why.
3. More tooltips could help to some degree.

To determine the best options / patterns we could do another round of supervised tests / interview based user studies during Carthage - from first visit to applying to a role for example.
